XWebView/XWVInvocation.swift

```diff
@@ -257,7 +257,7 @@ extension CVarArg {
         return _encodeBitsAsWords(self)
     }
 }
-extension Bool: CVarArg {
+extension Bool {
```

Comment:
This extension declaration should be removed if Bool type conforms CVarArg protocol in swift 4.2.